tensorflow/python/ops/image_ops_test.py

```diff
@@ -239,26 +239,31 @@ def testShapeInference(self):
 
 
 class AdjustGamma(test_util.TensorFlowTestCase):
-
-  def test_adjust_gamma_one(self):
-    """Same image should be returned for gamma equal to one"""
+  @test_util.run_deprecated_v1
+  def test_adjust_gamma_less_zero_float32(self):
+    """White image should be returned for gamma equal to zero"""
     with self.cached_session():
-      x_data = np.random.uniform(0, 255, (8, 8))
+      x_data = np.random.uniform(0, 1.0, (8, 8))
       x_np = np.array(x_data, dtype=np.float32)
 
       x = constant_op.constant(x_np, shape=x_np.shape)
-      y = image_ops.adjust_gamma(x, gamma=1)
 
-      y_tf = self.evaluate(y)
-      y_np = x_np
+      err_msg = "Gamma should be a non-negative real number."
 
-      self.assertAllClose(y_tf, y_np, 1e-6)
+      try:
+        image_ops.adjust_gamma(x, gamma=-1)
```

Comment:
Use self.assertRaises

Reply:
Fixed in commit [57835a8](https://github.com/tensorflow/tensorflow/pull/26508/commits/57835a8b9e9a19a061974d1fd393beeaf7898261)